lib/inspec/resources/ssh_config.rb

```diff
@@ -7,6 +7,7 @@ module Inspec::Resources
   class SshConfig < Inspec.resource(1)
     name "ssh_config"
     supports platform: "unix"
+    supports platform: "windows"
```

Comment:
This is a nice addition, thank you!

On lines 23 and 93 we refer to Posix filepaths. A quick read of Windows OpenSSH docs suggests the files will be in `%programdata%\ssh\`

It would be good to change

```
@conf_path = conf_path || "/etc/ssh/ssh_config"
```

to

```
@conf_path = conf_path || ssh_config_file("ssh_config")
```

where `ssh_config_file` is a new method that appends `%programdata%\ssh\` if Windows, else appends `/etc/ssh/`

Reply:
`file` doesn't seem to handle `%foo%`, so I had to go add a few more lines to retrieve the variable.